gkgen/bcp47gen_test.go

```diff
@@ -5,72 +5,70 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/stretchr/testify/suite"
+	"github.com/stretchr/testify/require"
 )
 
 type TestData struct {
 	BCP47String *string
 }
 
-// BCP47ValidatorTestSuite
-type BCP47ValidatorTestSuite struct {
-	suite.Suite
-}
-
-// TestBCP47ValidatorTestSuite
-func TestBCP47ValidatorTestSuite(t *testing.T) {
-	suite.Run(t, new(BCP47ValidatorTestSuite))
-}
-
 // TestIsBCP47_ParamsLength
-func (s *BCP47ValidatorTestSuite) TestIsBCP47_ParamsLength() {
+func TestIsBCP47_ParamsLength(t *testing.T) {
```

Comment:
I'd like to do this as part of making things table-driven (future PR) rather than going through and adding `t.Parallel()` into every test, if you're okay with that.